netkan/netkan/cli.py

```diff
@@ -101,14 +101,18 @@ def indexer(queue, metadata, token, repo, user, key,
     '--dev', is_flag=True, default=False,
     help='Disable AWS Credit Checks',
 )
-def scheduler(queue, netkan, max_queued, debug, dev):
+@click.option(
+    '--schedule-all', is_flag=True, default=False,
+    help='Schedule all modules even if we think they should be covered by webhooks'
+)
```

Comment:
Well it's more, web hooks can and do get missed. They are spray and pray by nature, So I think we should do full runs every now and then to ensure we don't miss stuff. How/When is definitely outside the scope of this PR.